app/scripts/projekt/settings/projektsettings.html

```diff
@@ -196,15 +196,18 @@ <h3 class="panel-title">
                   <label for="bezeichnung" translate>Mitteilung vor dem Login</label>
                 </div>
                 <div class="col-md-8">
-                      <textarea class="form-control" name="welcomeMessage1TextArea" ng-disabled="!editMode" ng-model="projekt.welcomeMessage1" placeholder=" {{'Mitteilung welche allen Nutzenden vor dem Einloggen angezeigt wird'|translate}}..." maxlength = 2000></textarea>
+                  <div class="welcomeMessageBorder" ng-if="!editMode" ng-bind-html="projekt.welcomeMessage1"></div>
+                  <ng-quill-editor class="form-control welcomeMessage1QuillEditor" name="welcomeMessage1TextArea" ng-if="editMode" ng-model="projekt.welcomeMessage1" bounds="self" placeholder="'Mitteilung welche allen Nutzenden vor dem Einloggen angezeigt wird'|translate" preserve-whitespace="false" track-changes="all" maxlength = 65000>
+                  </ng-quill-editor>
                 </div>
               </div>
               <div class="row form-group">
                 <div class="col-md-4">
                   <label for="welcomeMessage2Label" translate>Mitteilung nach dem Login</label>
                 </div>
                 <div class="col-md-8">
-                      <textarea class="form-control" name="welcomeMessage2TextArea" ng-disabled="!editMode" ng-model="projekt.welcomeMessage2" placeholder="{{'Mitteilung welche den Nutzenden nach dem erfolgreichen Einloggen angezeigt wird'|translate}}..." maxlength = 2000></textarea>
+                  <div class="welcomeMessageBorder" ng-if="!editMode" ng-bind-html="projekt.welcomeMessage2"></div>
+                  <ng-quill-editor class="form-control welcomeMessage1QuillEditor" name="welcomeMessage2TextArea" ng-if="editMode" ng-model="projekt.welcomeMessage2" bounds="self" placeholder="'Mitteilung welche den Nutzenden nach dem erfolgreichen Einloggen angezeigt wird'|translate" preserve-whitespace="false" track-changes="all" maxlength = 65000>
```

Comment:
Both form controls use the same class `welcomeMessage1QuillEditor` but there are two classes `welcomeMessage1QuillEditor` and `welcomeMessage2QuillEditor` defining the same attributes.
  1. If two are needed for some reason, use both `1` and `2` respectively
  2. Otherwise drop one and rename it to `welcomeMessageQuillEditor`

Reply:
you are right. Changing it right now